net-misc/igmpproxy/files/igmpproxy.initd-r1

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+config="/etc/${RC_SVCNAME}.conf"
+name="IGMPproxy"
+
+command="igmpproxy"
+command_args="${IGMPPROXY_OPTS} ${config}"
+procname="${RC_SVCNAME}"
+
+start_pre() {
```

Comment:
try using `required_files="/etc/file"` instead of custom function. probably should do exactly what you want.

Reply:
Great. I wasn't aware, such thing exists. Yes, it does what I want, so updated.